pyproject.toml

```diff
@@ -13,7 +13,9 @@ classifiers = [
 ]
 description = "Devices for tickit, an event-based device simulation framework"
 dependencies = [
-    "tickit"
+    "tickit",
+    "typing_extensions",
+    "softioc",
```

Comment:
How come softioc and typing_extensions are needed as direct dependencies? softioc is a tickit dependency so should be included .

Reply:
Not sure, but inspecting the container logs when built without this and running S03 showed me `module not found` errors so they weren't being included even if they should have been

Reply:
It would be nice to figure out why this doesn't work as expected, but I appreciate its not a priority so happy to leave it like this for now.